src/jit_compiler_a64.cpp

```diff
@@ -98,8 +101,10 @@ JitCompilerA64::JitCompilerA64()
 	memset(reg_changed_offset, 0, sizeof(reg_changed_offset));
 	memcpy(code, (void*) randomx_program_aarch64, CodeSize);
 
-#ifdef __GNUC__
-	__builtin___clear_cache(reinterpret_cast<char*>(code), reinterpret_cast<char*>(code + CodeSize));
+#ifdef HAVE_BUILTIN_CLEAR_CACHE
+ 	__builtin___clear_cache(reinterpret_cast<char*>(code), reinterpret_cast<char*>(code + CodeSize));
+#else
+ 	sys_icache_invalidate(code, (size_t)(code + CodeSize));
```

Comment:
The definition of the function is `sys_icache_invalidate(void *start, size_t len);` so it must be
```
sys_icache_invalidate(code, CodeSize);
```
here and in other places.

Reply:
Thanks, done.